lib/models/app_user_singleton.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:book/data/firebase_auth/firebase_auth_singleton.dart';
+import 'package:book/models/app_user.dart';
+
+class AppUserSingleton {
+  static AppUserSingleton? _instance;
+  AppUser? _appUser;
+
+  AppUserSingleton._internal() {
+    _appUser = _appUser;
```

Comment:
what does this mean?

Reply:
Im afraid all three of us are confused about this comment what's wrong with that part of the code?You wrote that code together with us.

Reply:
Ok, I will explain you in the officee